Framework/Algorithms/src/FilterEvents.cpp

```diff
@@ -128,69 +175,54 @@ void FilterEvents::init() {
   declareProperty("SpectrumWithoutDetector", "Skip", std::make_shared<StringListValidator>(spec_no_det),
                   "Approach to deal with spectrum without detectors. ");
 
-  declareProperty("SplitSampleLogs", true,
-                  "If selected, all sample logs will be splitted by the  "
-                  "event splitters.  It is not recommended for fast event "
-                  "log splitters. ");
+  declareProperty("DBSpectrum", EMPTY_INT(), "Spectrum (workspace index) for debug purpose. ");
 
-  declareProperty("NumberOutputWS", 0, "Number of output output workspace splitted. ", Direction::Output);
+  setPropertyGroup("FilterStartTime", titleAdditionalOptions);
+  setPropertyGroup("CorrectionToSample", titleAdditionalOptions);
+  setPropertyGroup("DetectorTOFCorrectionWorkspace", titleAdditionalOptions);
+  setPropertyGroup("IncidentEnergy", titleAdditionalOptions);
+  setPropertyGroup("SpectrumWithoutDetector", titleAdditionalOptions);
+  setPropertyGroup("DBSpectrum", titleAdditionalOptions);
 
-  declareProperty("DBSpectrum", EMPTY_INT(), "Spectrum (workspace index) for debug purpose. ");
+  //*********************
+  // Output Properties
+  //*********************
+  declareProperty("NumberOutputWS", 0, "Number of output output workspace splitted. ", Direction::Output);
 
   declareProperty(std::make_unique<ArrayProperty<string>>("OutputWorkspaceNames", Direction::Output),
                   "List of output workspaces names");
 
-  declareProperty("RelativeTime", false,
-                  "Flag to indicate that in the input Matrix splitting workspace,"
-                  "the time indicated by X-vector is relative to either run start time or "
-                  "some indicted time.");
-
-  declareProperty("FilterStartTime", "", "Start time for splitters that can be parsed to DateAndTime.");
-
-  declareProperty(std::make_unique<ArrayProperty<std::string>>("TimeSeriesPropertyLogs"),
-                  "List of name of sample logs of TimeSeriesProperty format. "
-                  "They will be either excluded from splitting if ExcludedSpecifiedLogs is "
-                  "specified as True. Or "
-                  "They will be the only TimeSeriesProperty sample logs that will be split "
-                  "to child workspaces.");
-
-  declareProperty("ExcludeSpecifiedLogs", true,
-                  "If true, all the TimeSeriesProperty logs listed will be "
-                  "excluded from duplicating. "
-                  "Otherwise, only those specified logs will be split.");
-
-  declareProperty("DescriptiveOutputNames", false,
-                  "If selected, the names of the output workspaces will "
-                  "include information about each slice.");
+  declareProperty("OutputUnfilteredEvents", false, "If selected, unfiltered events will be output.");
+  setPropertyGroup("OutputUnfilteredEvents", titleOutputWksp);
```

Comment:
These strings being copied everywhere reminds me of our discussion of a local `PropertyNames` namespace from the other day. Good times!

Reply:
Oh no!, not another Holy war :anguished: :exclamation: